doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -5600,7 +5600,7 @@ Coil:Cooling:DX:TwoStageWithHumidityControlMode,
    CoilPerformance:DX:Cooling,                 !- Normal Mode Stage 1 Coil Performance Object Type
    DOAS Standard Perf 1,                             !- Normal Mode Stage 1 Coil Performance Name
    CoilPerformance:DX:Cooling,                 !- Normal Mode Stage 1+2 Coil Performance Object Type
-   DOAS Standard Perf 1+2,                         !- Normal Mode Stage 1+2 Coil Performance Name
+   DOAS Standard Perf 1+2,                  When the fuel type is not electricity and the parent object AirLoopHVAC:UnitaryHeatPump:AirToAir:MultiSpeed does not require waste heat calculations, this field is ignored. If the field is blank, a warning will be issues and simulation continues. When the fuel type is not electricity and the parent object AirLoopHVAC:UnitaryHeatPump:AirToAir:MultiSpeed requires waste heat calculations, if this field is left blank, the program assumes a constant value of 1 to make simulation continue and a warning will be issued.        !- Normal Mode Stage 1+2 Coil Performance Name
```

Comment:
Something odd happened here.  But you can remove the dependency on fuel type here.